src/templates/base/base_chapter.html

```diff
@@ -149,6 +150,60 @@ <h2 id="explore-results">
   {% endif %}
 {% endmacro %}
 
+{% macro render_webmentions() %}
+  <h2 id="reactions">
+    <a href="#reactions" class="anchor-link">{{ self.reactions() }}</a>
+  </h2>
+  <div class="reactions">
+    <div class="reaction-tabs" role="tablist" aria-label="Reaction Tabs">
+      <button
+        role="tab"
+        aria-selected="true"
+        aria-controls="likes-panel"
+        id="likes-tab"
+        tabindex="0"
+      >
+        <span id="likes-count">0</span> {{ self.likes() }}
```

Comment:
I think we'd need additional logic to handle singular and plural grammar, ie "1 like" vs "2 likes".

This is complicated for translations that follow different rules than simply 1 or not-1. @tunetheweb do we have any precedent for how to handle that in the translations?

Reply:
Yup:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/4b332cae91268b62fa5d208a5bc681a88242add6/src/static/js/almanac.js#L471-L479

Though suspect we need more than that for Slavic languages that have more than just singlar and plural. We have a `plural_ru` macro for that but looks like we never implemented that for the comment/comments. Can look at that later.

Reply:
Ah ok. Let's get basic singular/plural support into this PR then we can worry about the more complex rules in a separate issue.

I also just came across [`Intl.PluralRules.select()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules/select) which looks like it could useful.